[2021-02-18 22:51:04,808] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: tweets_by_user.get_tweets_by_user_list 2021-02-18T22:50:00+00:00 [queued]>
[2021-02-18 22:51:04,814] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: tweets_by_user.get_tweets_by_user_list 2021-02-18T22:50:00+00:00 [queued]>
[2021-02-18 22:51:04,814] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2021-02-18 22:51:04,814] {taskinstance.py:1018} INFO - Starting attempt 1 of 2
[2021-02-18 22:51:04,814] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2021-02-18 22:51:04,824] {taskinstance.py:1038} INFO - Executing <Task(PythonOperator): get_tweets_by_user_list> on 2021-02-18T22:50:00+00:00
[2021-02-18 22:51:04,826] {standard_task_runner.py:51} INFO - Started process 19637 to run task
[2021-02-18 22:51:04,832] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'tweets_by_user', 'get_tweets_by_user_list', '2021-02-18T22:50:00+00:00', '--job-id', '1165', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/tweets_by_user.py', '--cfg-path', '/tmp/tmp2hht8i_h']
[2021-02-18 22:51:04,834] {standard_task_runner.py:76} INFO - Job 1165: Subtask get_tweets_by_user_list
[2021-02-18 22:51:04,863] {logging_mixin.py:103} INFO - Running <TaskInstance: tweets_by_user.get_tweets_by_user_list 2021-02-18T22:50:00+00:00 [running]> on host ip-172-31-3-59.ec2.internal
[2021-02-18 22:51:04,890] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=airflow@example.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=tweets_by_user
AIRFLOW_CTX_TASK_ID=get_tweets_by_user_list
AIRFLOW_CTX_EXECUTION_DATE=2021-02-18T22:50:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-02-18T22:50:00+00:00
[2021-02-18 22:51:04,891] {logging_mixin.py:103} INFO - Autenticação no twitter
[2021-02-18 22:51:04,906] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:51:05,099] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:51:05,698] {logging_mixin.py:103} INFO - 200
[2021-02-18 22:51:05,712] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:51:05,865] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:51:06,338] {logging_mixin.py:103} INFO - 400
[2021-02-18 22:51:06,346] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:51:06,501] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:51:06,876] {logging_mixin.py:103} INFO - 600
[2021-02-18 22:51:06,879] {logging_mixin.py:103} INFO - 0:00:01.988727
[2021-02-18 22:51:06,879] {logging_mixin.py:103} INFO - 600
[2021-02-18 22:51:08,846] {logging_mixin.py:103} INFO - 0:00:01.966208
[2021-02-18 22:51:08,889] {logging_mixin.py:103} INFO - Autenticação no banco de dados twitter_data
[2021-02-18 22:51:08,910] {logging_mixin.py:103} INFO - 0:00:00.064094
[2021-02-18 22:51:08,910] {logging_mixin.py:103} INFO - 600 devidamente inseridas no banco
[2021-02-18 22:51:08,912] {python.py:118} INFO - Done. Returned value was: None
[2021-02-18 22:51:08,915] {taskinstance.py:1142} INFO - Marking task as SUCCESS. dag_id=tweets_by_user, task_id=get_tweets_by_user_list, execution_date=20210218T225000, start_date=20210218T225104, end_date=20210218T225108
[2021-02-18 22:51:08,936] {taskinstance.py:1195} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-02-18 22:51:08,977] {local_task_job.py:118} INFO - Task exited with return code 0
